src/nl/rubensten/texifyidea/window/PreviewForm.kt

```diff
@@ -0,0 +1,126 @@
+                        """
+\end{document}"""
+                writer.println(tmp_content)
+                writer.close()
+
+                val latex = Runtime.getRuntime().exec(
+                        arrayOf("pdflatex",
+                                "-halt-on-error",
+                                tmp_basename + ".tex",
+                                "-interaction=nonstopmode"),
+                        null,
+                        tmpdir
+                )
+                latex.outputStream.close()
+                if (!latex.waitFor(3, TimeUnit.SECONDS)){
+                    latex.destroy()
+                    output_area!!.text = "Latex took more than 3 seconds. Terminated."
+                    preview_panel!!.clear_image()
+                    return;
+                }
+
+                val reader = BufferedReader(InputStreamReader(latex.inputStream))
+                output_area!!.text = reader.readText()
+                reader.close()
+                if (latex.exitValue() == 0) {
+                    val pdf2svg = Runtime.getRuntime().exec(
+                            arrayOf("pdf2svg",
+                                    tmp_basename + ".pdf",
+                                    tmp_basename + ".svg"
+                            ),
+                            null,
+                            tmpdir
+                    )
+
+                    if (!pdf2svg.waitFor(3, TimeUnit.SECONDS)){
+                        pdf2svg.destroy()
+                        preview_panel!!.clear_image()
+                        output_area!!.text = "pdf2svg took more than 3 seconds. Terminated."
+                        return;
+                    }
+
+                    if (pdf2svg.exitValue()==0){
+
+                        val inkscape = Runtime.getRuntime().exec(
+                                arrayOf("inkscape",
+                                        tmp_basename + ".svg",
+                                        "--export-area-drawing",
+                                        "--export-dpi", "1000",
+                                        "--export-background", "#FFFFFF",
+                                        "--export-png", tmp_basename + ".png"
+                                ),
+                                null,
+                                tmpdir
+                        )
+                        if (!inkscape.waitFor(3, TimeUnit.SECONDS)){
+                            inkscape.destroy()
+                            preview_panel!!.clear_image()
+                            output_area!!.text = "inkscape took more than 3 seconds. Terminated."
+                            return;
+                        }
+
+                        if (inkscape.exitValue()==0){
+                            val image = ImageIO.read(File(tmp_basename + ".png"))
+
+                            preview_panel!!.set_image(image)
+                            preview_panel!!.requestFocus()
+                        }else{
```

Comment:
```suggestion
                        }
                        else {
```
Else on new line, see [code conventions](https://github.com/Ruben-Sten/TeXiFy-IDEA/blob/master/CONTRIBUTING.md#code-conventions)